SLACK.md

```diff
@@ -0,0 +1,94 @@
+# <img src="img/slack_logo.png" alt="Slack" width="300px">
```

Comment:
It's unfortunate, but I decided to use an `img` tag here because I wanted the Slack logo to look crisp on retina displays, but I couldn't find an SVG of their logo and I can't downscale images using default Markdown.